### DISCUSSION TITLE
razorpay payment not working without order id (razorpay order id)

### DISCUSSION THREAD

**Discussion by itzmanish** [saleor/saleor#7213] [General] [answered]
I am trying to implement razorpay PSP but the client side needs an order id which should be created on razorpay.
How to fix that?

**Comment by karolkielecki** [comment, marked as answer]
@itzmanish Hi! I have just replied to your Spectrum thread. https://spectrum.chat/saleor/general/razorpay-payment-gateway-needs-to-have-an-order-id-otherwise-it-fails~86c1a326-93f7-470c-9ec8-214687661f0d?m=MTYxODk5OTMyNjc4Ng==
